### PR TITLE
Bug/828 slash at end of uri path

### PR DESCRIPTION
**Pull request by kzangeli**
Implemnets #828 

The title says is all.
The path of the URL of incoming request are stripped of trailing '/' before continuing.
This fixes the issue 828.

### CODE REVIEW

File: src/lib/rest/rest.cpp [hunk 1]
```diff
@@ -604,7 +604,7 @@ static int contentTypeCheck(ConnectionInfo* ciP)
```
Comment (fgalan) [inline src/lib/rest/rest.cpp]: CHANGES_NEXT_RELEASE entry missing?

Comment (kzangeli) [inline src/lib/rest/rest.cpp]: Ah, of course!
When the branch was prepared (all these queued PR-branches), develop wasn't ready and I couldn't touch CHANGES_NEXT_RELEASE. I had completely forgotten ...

Comment (kzangeli) [inline src/lib/rest/rest.cpp]: Fixed in c4fc504

**Comment by fgalan** [comment]
LGTM

**Pull request merged**